### DISCUSSION TITLE
UMI

### DISCUSSION THREAD

**Discussion by PavithraV0223** [milaboratory/mixcr#1409] [Q&A] [answered]
## Checklist before submitting the issue:

 - [x] The issue is strongly related to the MiXCR software
 - [x] The issue can be reproduced with [the most recent version](https://github.com/milaboratory/mixcr/releases) of MiXCR
 - [x] There is no answer to the question in the [official documentation](http://mixcr.readthedocs.io/) and there is no duplicate issue in the [bug tracker](https://github.com/milaboratory/mixcr/issues?utf8=✓&q=)  
 - [x] Inspection of raw alignments with [exportAlignmentsPretty](http://mixcr.readthedocs.io/en/master/export.html#exporting-well-formatted-alignments-for-manual-inspection) shows that data has the expected architecture, and sample preparation artefacts are not the reason of the problem (if this is the matter of the issue)

## Expected Result

*A description of what you wanted to happen*

## Actual Result

*A description of what actually happened*

## Exact MiXCR commands 

*Paste here __exact__ MiXCR commands you used (enclose it with single `` ` ``  for better formatting)*

## MiXCR report files 

*Paste here content of the report files produced by MiXCR (enclose it with triple ```` ``` ````  for better formatting)*

**Comment by mizraelson** [comment]
Hi. Nanopore sequencing is usually unoriended, which means UMI can be on either end of the read. Can you share the library structure (are there any barcode sequences present that we can use as an anchor) and maybe a file with a few reads so I can adjust the pattern? 

Sincerely,
Mark

**Comment by mizraelson** [comment, marked as answer]
Sorry for the delay.
Please try the command bellow:

```
mixcr analyze generic-ont-with-umi \
	--species lamaGlama \
	--tag-pattern "^*CTTCCGATCT(UMI:N{12})N{5}(R1:*)" \
	-Malign.tagUnstranded=true \
	data.fastq \
	output
```

Let me know how it worked.

Sincerely,
Mark

**Comment by PavithraV0223** [comment]
Thank you, It did work
